RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelRawToClusterGPUKernel.cu

```diff
@@ -355,12 +355,12 @@ namespace pixelgpudetails {
       pixelgpudetails::DetIdGPU detId = getRawId(cablingMap, fedId, link, roc);
       uint32_t rawId = detId.rawId;
       uint32_t layer = 0;
-      int side = 0, panel = 0, module = 0;
+      int side = 0, panel = 0, module_ = 0;
```

Comment:
trailing underscores are preferred for class member names; this can be confusing. Please use some other name, e.g. `aModule`